plugins/airbrake-backend/knip-report.md

```diff
@@ -0,0 +1,10 @@
+# Knip report
+
+## Unused devDependencies (3)
+
+| Name             | Location     |
+|:-----------------|:-------------|
+| supertest        | package.json |
```

Comment:
There are a couple of false negatives I think - this one is clearly used in a test for example. Not sure why this is being missed.

Reply:
Any idea about these @webpro?

Reply:
Sorry, but I shouldn't be going into codebases for one-offs. Please open an issue in the Knip repo if you think there are false positives.